server/plugin/user_test.go

```diff
@@ -190,25 +191,20 @@ func Test_InitOAuth2(t *testing.T) {
 		loadUserError error
 	}{
 		{
-			description:   "User is already connected to ServiceNow",
-			errMessage:    nil,
-			expectedErr:   "mockErrMessage",
-			loadedUser:    &serializer.User{},
-			loadUserError: nil,
+			description: "User is already connected to ServiceNow",
+			expectedErr: "user is already connected to ServiceNow",
+			loadedUser:  &serializer.User{},
 		},
 		{
 			description:   "OAuth2 is initialized successfully",
-			errMessage:    nil,
 			expectedErr:   "",
-			loadedUser:    nil,
-			loadUserError: errors.New("mockErrMessage"),
+			loadUserError: errors.New("user is not present in KVstore"),
 		},
 		{
 			description:   "Error occurred while storing oauth2 state",
-			errMessage:    errors.New("mockErrMessage"),
-			expectedErr:   "mockErrMessage",
-			loadedUser:    nil,
-			loadUserError: errors.New("mockErrMessage"),
+			errMessage:    errors.New("error storing OAuth2 state"),
+			expectedErr:   "error storing OAuth2 state",
+			loadUserError: errors.New("user is not present in KVstore"),
```

Comment:
Should this error be present in this test case?

Reply:
Yes, as if user is not present in Kv store it will give error and we will connect the user but if it is present that means user is already connected

Reply:
If we don't return a error from LoadUser our function will not reach the storeUser function